draft-ietf-mls-architecture.md

```diff
@@ -333,39 +355,13 @@ impersonate group members. We discuss security considerations
 regarding the compromise of the different AS functionalities in detail
 in Section {{as-compromise}}.
 
-### Credential Authentication
-
-In many cases, the first functionality might be provided by a service
-which fulfills a role similar to a certification authority in the
-WebPKI: it provides a binding of an identity (e.g., a user name, phone
-number, email address, etc) to a signature key. The identity/signature
-key pair can then either be used directly in a group, or as an root of
-trust which in turn authenticates credentials used in the group.
-
-The flexibility afforded by the latter option allows for multiple
-infrastructure considerations and has the benefit of providing ways to
-use different signature keys across different groups by using
-hierarchical authentication keys. This flexibility also comes at the
-price of a security tradeoff, described in the security
-considerations, between potential unlinkability of the signature keys
-across groups and the amount of time required to reinstate
-authentication and secrecy of messages after the compromise of a
-device.
-
-### Message Authentication
-
-MLS messages are authenticated by a signature conforming to the
-signature scheme of the group's ciphersuite. To allow for message
-deniability (see Section {{Non-Repudiation-vs-Deniability}}), messages
-are not required to be signed by the private key corresponding to a
-member's credential, but the key must be authenticated using some
-mechanism. Thus, message authentication relies on the accuracy of
-the key's authentication vice the credential authentication.
-
-While credential authentication can be performed by a separate entity,
-message authentication should be performed by each member separately
-due to the encryption layer of the protocol which protects the
-signature of the message.
+The association between members' identities and signature keys is fairly
+flexible in MLS.  As noted above, there is no requirement that all appearances
```

Comment:
```suggestion
flexible in MLS.  As noted above, there is no requirement that all clients
```

Reply:
As noted in #85, the spec doesn't allow this as long as each client is represented by its own leaf.

Reply:
In principle, it is possible, as long as the two clients share their entire cryptographic state.  Obviously, that's a lot more challenging than just sync'ing a signature key, but might still be feasible?

Reply:
Then, from an MLS standpoint, it's really the same client 🤷🏻‍♂️.